### DISCUSSION TITLE
Not able navigate to components page on development server.

### DISCUSSION THREAD

**Discussion by ShrinidhiUpadhyaya** [rsuite/rsuite#3749] [Q&A]
I have cloned the repository and have followed the contribution guide to build it.
```
 npm i && npm i --prefix docs
 npm run dev --prefix docs
```
I am able to start the development server and the homepage loads, but when i click on **_'Components'_** i am not able to navigate to the components page, it infinitely keeps on compiling. Could someone help me out on this?

![image](https://github.com/rsuite/rsuite/assets/8769408/064ba408-ed62-4ac0-8169-075b6a945e8a)
![image](https://github.com/rsuite/rsuite/assets/8769408/775a0d3d-1e1f-478f-b24d-30cebfa5568f)

**Comment by simonguo** [comment]
Hi @ShrinidhiUpadhyaya.

Because there are too many files to compile, the first time Nextjs builds in the development environment is very slow, you need to wait patiently.
<img width="578" alt="image" src="https://github.com/rsuite/rsuite/assets/1203827/fe7f0aaf-a8e1-4308-8e20-f9034b686a16">

**Comment by ShrinidhiUpadhyaya** [comment]
Hi @simonguo , no actually what i meant is i have waited twice for almost half an hour, it doesn't load at all.

OS:- Windows 11

![image](https://github.com/rsuite/rsuite/assets/8769408/7b9f296e-c92e-49a0-8573-9d96954fe73a)

**Reply by simonguo** [reply]
You try to configure `swcMinify` in `docs/next.config.js`.

```
module.exports = {
  ...
  swcMinify: true,
}
```

If you still can't solve your problem, I will upgrade nextjs to the latest version when I take the time, and there will be a certain performance improvement.

**Reply by simonguo** [reply]
https://github.com/rsuite/rsuite/pull/3753

I upgraded next to v14, you can try the `build/nextjs-14` branch.

**Comment by ShrinidhiUpadhyaya** [comment]
Thank you so much the `build/nextjs-14` works fine.
